### PR TITLE
SOME CHANGES FOR THE MASTER

### PR DESCRIPTION
**Pull request by stavroulapap**
with fork i have your repository
in my repository,i didnt add the changes to the master. Changes are on the first branch
the first branch has the changes that i suggest you to put in your master

### CODE REVIEW

**Review by stavtesting — APPROVED** [review]

**Review by stavtesting — COMMENTED** [review]
its ok but...

File: testing1.txt [hunk 1]
```diff
@@ -1,15 +1,19 @@
-hello world
+hello stavroula
```
Comment (stavtesting) [inline testing1.txt:1]: δεν το θελω ετσι, θελω το hello world

File: testing2.txt [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
+taylor swift
 
-tate
+
+mcrae tate
 
 katy
+perry
+
+john b
```
Comment (stavtesting) [inline testing2.txt:9]: θελω τον jj οχι τον john b

**Review by stavtesting — APPROVED** [review]

**Comment by stavroulapap** [comment]
τελεια

**Pull request merged**